packages/rln/src/keystore/keystore.ts

```diff
@@ -250,32 +251,32 @@ export class Keystore {
       const str = bytesToUtf8(bytes);
       const obj = JSON.parse(str);
 
-      // TODO: add runtime validation of nwaku credentials
+      // Get identity fields from named object
+      const { idTrapdoor, idNullifier, idSecretHash, idCommitment } = _.get(
+        obj,
+        "identityCredential",
+        {}
+      );
+
+      const idTrapdoorArray = new Uint8Array(idTrapdoor || []);
```

Comment:
does this conversion work? I remember there were issues so that `fromArraylikeToBytes` and `Array.from(options.identity.IDTrapdoor)` were used